.github/workflows/ci.yml

```diff
@@ -27,6 +27,7 @@ jobs:
           - command: test --no-default-features --locked --all
           - command: test --all-features --locked --all
           - command: doc --all-features --locked --no-deps
+          - command: bench --locked --all
```

Comment:
I don't think we should run the benches in CI. Lets just build them so they compile check

Reply:
ahh, of course. done.